src/tests/ftest/io/nvme_object.py

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2019 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+"""
+from __future__ import print_function
+
+import threading
+import avocado
+
+from daos_api import DaosApiError
+from test_utils import TestPool, TestContainer
+from apricot import TestWithServers
+
+
+def container_write(container, record):
+    """Method to write to a container
+
+       Args:
+         container: instance of TestContainer
+         record: record size to be written
+    """
+    # update record qty
+    container.record_qty.update(record)
+    print("\nRecord Size:{}".format(container.record_qty))
+
+    # write multiple objects
+    container.write_objects()
+
+    # read written objects and verify
+    container.read_objects()
```

Comment:
write_object() is all ready doing this [code link](https://github.com/daos-stack/daos/blob/311ea2ef54dd8c4d2444c753c91354a2450ecfc0/src/tests/ftest/util/test_utils.py#L703) 

Can we separate this and have different container_read()? 
So flow will be start all container.write_object(), do join() and wait till all write finish. Later call container.read_object() threads, do join and wait for all read threads to finish. 
write_object test code might be generating the random data so not sure if we can use that instead we can use write_record().

Reply:
Done